package.json

```diff
@@ -20,6 +20,7 @@
     "astro-icon": "^0.8.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "sass": "^1.69.4",
```

Comment:
Why have you added SASS as a dependency?

Reply:
to add the hover effect of View in Detail button
As it was a component I copied from internet which required SASS

Reply:
Please use a SASS to CSS transpiler to convert it to CSS and then add those styles. Adding SASS for just one component is not worth the pain in building.

Reply:
please check